packages/react/src/components/ComboBox/ComboBox-test.e2e.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import 'carbon-components/scss/components/combo-box/_combo-box.scss';
+import 'carbon-components/scss/components/list-box/_list-box.scss';
+import 'carbon-components/scss/components/text-input/_text-input.scss';
+
+import React from 'react';
+import { mount } from '@cypress/react';
+import ComboBox from './ComboBox';
+
+describe('ComboBox', () => {
+  beforeEach(() => {
+    const items = [
+      {
+        id: 'option-0',
+        text:
+          'An example option that is really long to show what should be done to handle long text',
+      },
```

Comment:
Forgot to mention something I found useful for Dropdown was the `generateItems` and `generateGenericItem` helpers from `../ListBox/test-helpers`. It helps cut down the boilerplate a bit